pinot-plugins/pinot-file-system/pinot-adls/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1,4 @@
+# This class defines Mockito plugins to be used during testing
+
+# mock-maker inline enables mocking of 'final' classes in Mockito v2
+mock-maker-inline
```

Comment:
Missing an empty line at the end of the file

Reply:
Also why is this file needed?

Reply:
Most of the classes in the azure sdk are 'final', they cant be mocked without this declaration.

I updated the empty line in the code base, github ignores it for some reason.